system-addon/content-src/components/LinkMenu/LinkMenu.jsx

```diff
@@ -0,0 +1,45 @@
+const React = require("react");
+const ContextMenu = require("content-src/components/ContextMenu/ContextMenu");
+
+class LinkMenu extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {visible: false};
+    this.getOptions = this.getOptions.bind(this);
+  }
+  getOptions() {
```

Comment:
I'm trying to figure out why this is a function here instead of just a  constant